packages/dev/core/src/FlowGraph/Blocks/Data/flowGraphAddBlock.ts

```diff
@@ -0,0 +1,15 @@
+import type { FlowGraph } from "../../flowGraph";
+import { FlowGraphBinaryOpBaseBlock } from "./flowGraphBinaryOpBlock";
+
+/**
+ * @experimental
+ */
+export class FlowGraphAddNumbersBlock extends FlowGraphBinaryOpBaseBlock<number, number, number> {
+    constructor(graph: FlowGraph) {
+        super(graph, {
+            defaultLeftValue: 0,

```

Comment:
why the name defaultXXX, can we just use XXX like left and right?

Reply:
I was on the same topci for NGE. the value is the value returned unless there is a connected value. I was tempted initially to call them notConnectedValue but it is dumb :)